internal/service/alarms/internal/alarms_server.go

```diff
@@ -57,12 +60,10 @@ func (a *AlarmsServer) GetAlarms(ctx context.Context, request api.GetAlarmsReque
 // GetAlarm returns an AlarmEventRecord with a given ID
 func (a *AlarmsServer) GetAlarm(ctx context.Context, request api.GetAlarmRequestObject) (api.GetAlarmResponseObject, error) {
 	aerModel, err := a.AlarmsRepository.GetAlarmEventRecordWithUuid(ctx, request.AlarmEventRecordId)
-	if err != nil {
+	if !errors.Is(err, pgx.ErrNoRows) {
```

Comment:
pgx.ErrNoRows is not returned but the custom error you created in common/utils/repository